DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -181,14 +181,19 @@ public List<DelegateUser> GetDelegateUsersByEmailAddress(string emailAddress)
         {
             List<DelegateUser> users = connection.Query<DelegateUser>(
                 @"SELECT
-                        CandidateID,
-                        FirstName,
-                        LastName,
-                        EmailAddress,
-                        Password,
-                        ResetPasswordID
-                    FROM Candidates
-                    WHERE (EmailAddress = @emailAddress)",
+                        cd.CandidateID AS Id,
+                        cd.CandidateNumber,
+                        cd.CentreID,
+                        ct.CentreName,
+                        ct.Active AS CentreActive,
+                        cd.EmailAddress,
+                        cd.FirstName,
+                        cd.LastName,
+                        cd.Password,
+                        cd.Approved
+                    FROM Candidates AS cd
+                    INNER JOIN Centres AS ct ON ct.CentreID = cd.CentreID
+                    WHERE cd.EmailAddress = @emailAddress",

```

Comment:
This should have data tests for on the latest master - do you mind double checking that everything gets asserted properly? 🙏